pkg/v1/providers/yttcb/clusterbootstrap.yaml

```diff
@@ -181,7 +206,7 @@ metadata:
   annotations:
     tkg.tanzu.vmware.com/add-missing-fields-from-tkr: #@ data.values.KUBERNETES_RELEASE
 spec:
-  #@ if data.values.CNI == "antrea":
+  #@ if data.values.CNI == "antrea" and antrea_configs_exist():
```

Comment:
Thanks for bringing this up. If the antrea configs are provided but `CNI: antrea` is not, I guess the AntreaConfig will be generated but not ClusterBootstrap.spec.cni?

It seems that we need to add more judgement conditions regarding no `CNI: antrea` param...

Reply:
yes, that will happen.
would like to make pipeline happy then probably narrow down the enhancement in this PR, hope this makes sense.

Reply:
Agree